site/app/libraries/plagiarism/Interval.php

```diff
@@ -23,6 +26,10 @@ public function getStart(): int {
         return $this->start;
     }
 
+    public function getType(): string {
+        return $this->type;
+    }
```

Comment:
A usage of this function should be added to `site/tests/app/libraries/plagiarism/IntervalTester.php`